tests/wake-format/basic/stdout

```diff
@@ -585,3 +585,19 @@ def defaultWake Unit =
     Pass (Pair "tmp/bin/wake" wakeVisible)
 
 
+def (
```

Comment:
One thing to watch out for since we're changing the CST here in a non-whitespace way, is that we're idempotent. That is if we see a long paren with a series of apply we format it the same way as a long series of applies in a pattern without this. I think we're consistent on this front right now but we should add a two tests

1) `def (tool (addHeaders: String) ... (addHeaders: String))`
2)
```
def (
  tool
  (addHeaders: String)
  ...
  (addHeaders: String)
)
```

This is to smoke test that we're not going to wind up adding parens to something that already gas parens